test/spec/waitFor.js

```diff
@@ -7,6 +7,19 @@ describe('waitFor', () => {
                 .should.be.equal(true)
         })
 
+        it('should pass fast with a short waitfor interval', async function () {
+            (await this.client.waitForEnabled('//html/body/section/input[8]', duration, 20))
+                .should.be.equal(true)
+        })
+
+        it('should timeout with a long waitfor interval', async function () {
+            try {
+                (await this.client.waitForEnabled('//html/body/section/input[8]', duration, 20000))
```

Comment:
This would make the test case wait for 20s right? Let's choose a smaller number to not blow up the test duration.